src/main/java/org/example/controller/DeveloperController.java

```diff
@@ -3,28 +3,42 @@
 import org.example.constant.ContextType;
 import org.example.constant.JspPage;
 import org.example.constant.ResponseCode;
+import org.example.constant.UrlPath;
+import org.example.converter.ArgumentToDeveloperConverter;
 import org.example.model.Arguments;
 import org.example.model.Developer;
 import org.example.model.JspAttribute;
 import org.example.model.Response;
 import org.example.service.DeveloperService;
 import org.example.service.JsonService;
+import org.example.web.PathMapper;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.util.ArrayList;
 import java.util.List;
 
 public class DeveloperController {
-
     private static final Logger LOGGER = LoggerFactory.getLogger(DeveloperController.class);
     private final DeveloperService developerService = DeveloperService.getInstance();
     private final JsonService jsonService = JsonService.getInstance();
+    private final ArgumentToDeveloperConverter argumentToDeveloperConverter = ArgumentToDeveloperConverter.getInstance();
+    private static DeveloperController instance;
 
+    private DeveloperController() {
+        PathMapper pathMapper = PathMapper.getInstance();
 
-    private static DeveloperController instance;
+        pathMapper.addMapping(UrlPath.GET_DEVELOPERS_JSON, this::getJsonPage);
+        pathMapper.addMapping(UrlPath.GET_ALL_DEVELOPERS_HTML, this::getTablePage);
+        pathMapper.addMapping(UrlPath.POST_DEVELOPERS_HTML, this::createDeveloperHtml);
+        pathMapper.addMapping(UrlPath.POST_DEVELOPERS_JSON, this::createDeveloperJson);
+    }
+
+    public DeveloperController init(){
```

Comment:
Why do we need to return something from this method?
What is the responsibility of it?